.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -3534,7 +3534,7 @@ stages:
     - job: upload
       timeoutInMinutes: 60 #default value
       pool:
-        vmImage: ubuntu-20.04
+        vmImage: ubuntu-latest
```

Comment:
FWIW, I would have switched everything to `ubuntu-22.04` instead of `ubuntu-latest` due to war wounds with things changing underneath us, but thanks for fixing this 😄